taskana-adapter-wildfly-example/pom.xml

```diff
@@ -1,30 +1,22 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
remove all unnecessary comments?

Reply:
I would prefer to leave those comments that explain why we do something, e.g. why we don't read properties from a file.